### PR TITLE
Add async restore queue

### PR DESCRIPTION
**Pull request by proteusvacuum**
https://github.com/dimagi/commcare-hq/pull/12251
@snopoke @sravfeyn

### CODE REVIEW

**Comment by proteusvacuum** [comment]
@benrudolph

File: fab/services/templates/supervisor_celery_async_restore_queue.conf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+directory={{ code_current }}
+user={{ sudo_user }}
+numprocs=1
+autostart=true
+autorestart=true
+stopasgroup=true
+killasgroup=true
+stdout_logfile={{ log_dir }}/celery_async_restore_queue.log
+redirect_stderr=true
+stderr_logfile={{ log_dir }}/celery_async_restore_queue.log.log
```
Comment (benrudolph) [inline fab/services/templates/supervisor_celery_async_restore_queue.conf]: did you mean this to `error.log`?

**Comment by benrudolph** [comment]
👍 minor comment

File: fab/services/templates/supervisor_celery_async_restore_queue.conf (hunk 1 above)
Comment (proteusvacuum) [inline fab/services/templates/supervisor_celery_async_restore_queue.conf]: oops, yes

**Pull request merged**